src/Target.h

```diff
@@ -311,6 +311,9 @@ struct Target {
      * Target. This is 2^31 - 1 except on 64-bit targets when the LargeBuffers
      * feature is enabled, which expands the maximum to 2^63 - 1. */
     int64_t maximum_buffer_size() const {
+        if (has_feature(Target::HexagonDma)) {
```

Comment:
This shouldn't be necessary, Hexagon is 32-bit. Can we just ignore this for the mock DMA implementation?

Reply:
Agreed.